src/domain/Lists/DraggableList/DraggableList.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import {
+  DragDropContext,
+  Draggable,
+  DropResult,
+  Droppable
+} from 'react-beautiful-dnd'
+import uuid from 'uuid'
+
+import { Props } from '../../../common'
+import { StyledListArea, StyledListItem } from './style'
+
+interface IDraggableListProps<T> {
+  /** The array of items */
+  items: T[]
+  /** Function to update the array of items when they have been re-arranged */
+  setItems: (nodes: T[]) => void
```

Comment:
I think you should be able to import a type for this?
Assuming that `setItems` will be a useState setter